CWE-94-Python_darwin/GAIR-NLP_PC-Agent/fs.py

```
@@ -0,0 +1,78 @@
+
+
+def hide_folder(folder_path):
+    # Set folder attribute to hidden
+    FILE_ATTRIBUTE_HIDDEN = 0x02
+    ctypes.windll.kernel32.SetFileAttributesW(folder_path, FILE_ATTRIBUTE_HIDDEN)
+
+
+def set_hidden_file(file_path, hide=True):
+    # Set or remove hidden attribute
+    if hide:
+        os.system(f'attrib +h "{file_path}"')
```

Ruff (0.13.2)

Starting a process with a shell, possible injection detected

(S605)